docs/Pflichtenheft.tex

```diff
@@ -419,19 +419,21 @@ \subsubsection{Änderung der Zu-/Abflussgeschwindigkeit}
 % \item [Nichtfunktionale Anforderungen] Die geänderten Einstellungen werden binnen einer halben Sekunde in der \gls{GUI} dargestellt.
 %\end{description}
 
-\subsubsection{Änderung der Aktualisierungsfrequenz}
+\subsubsection{Änderung der Verbindungseinstellungen}
 \begin{description}
-  \item[Name] Erhöhen oder Verringern der Aktualisierungsfrequenz.
+  \item[Name] Verändern der Verbindungseinstellungen.
   \item[Akteure] Manuel (Bediener der Software)
   \item[Eingangsbedingung] Die \gls{Fertigungssimulation} und die \gls{Uberwachungskonsole} sind in Betrieb und miteinander verbunden.
   \item[Ereignisfluss]~\\
   \begin{itemize}[noitemsep]
-    \item Manuel öffnet in der \gls{Uberwachungskonsole} die Einstellungen.
-    \item Manuel navigiert in den Reiter "`Allgemeine Einstellungen"'.
-    \item Manuel verschiebt den Schieberegler, der die Aktualisierungsfrequenz einstellt.
-    \item Manuel klickt den "`Speichern"' Knopf.
+    \item Manuel öffnet in der \gls{Uberwachungskonsole} die allgemeinen Einstellungen.
+    \item Manuel stellt die Aktualisierungsfrequenz, die \gls{IP-Adresse} oder einen Port der \gls{OPC UA Server} ein.
+    \item Manuel klickt den "`Speichern"' Knopf und das Einstellungsfenster schließt sich.
+    \item Der Verbindungsaufbau schlägt fehlt und das Einstellungsfenster öffnet sich wieder.
```

Comment:
Ja klar, erst wenn man auf "Speichern" (so heißt es glaub ich in den GUI Entwürfen) drückt, wird die Verbindung aufgebaut und das Einstellungsfenster geschlossen. Und wenn es fehltschlägt, geht das Einstellungsfenster halt wieder auf.